assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -115,6 +116,19 @@ export default function SettingsForm( {
 				gtmAnalyticsPropertyID={ analyticsSinglePropertyID }
 			/>
 
+			{ ga4ReportingEnabled && (
+				<div className="googlesitekit-settings-module__fields-group googlesitekit-settings-module__fields-group--no-border">
+					<h4 className="googlesitekit-settings-module__fields-group-title">
+						{ __( 'Dashboard view', 'google-site-kit' ) }
+					</h4>
+					<div className="googlesitekit-settings-module__meta-item googlesitekit-settings-module__meta-item--dashboard-view">
+						{ isGA4Connected && <GA4DashboardViewToggle /> }
+						{ ! isGA4Connected &&
+							__( 'Universal Analytics', 'google-site-kit' ) }
```

Comment:
Shouldn't this _also_ appear when GA4 is connected? With GA4 connected the screen looks like this:

<img width="995" alt="CleanShot 2023-03-22 at 12 09 09" src="https://user-images.githubusercontent.com/90871/226900755-4bae50a5-7483-4415-afbf-54ab109d2bb3.png">

And without it connected we get the header when only UA is connected:



Which is kind of odd as there's a heading for GA4 but not UA. I'd think we want:

<img width="926" alt="CleanShot 2023-03-22 at 12 12 15" src="https://user-images.githubusercontent.com/90871/226901378-5949d1fb-9284-4a28-86a2-bd4a4edc0b96.png">

That said, that implementation looks off compared to the GA4 header.

We'd likely want to add this markup to https://github.com/google/site-kit-wp/blob/01bd910aef4fa473c7dee60d85e2d71024f85fcc/assets/js/modules/analytics/components/settings/SettingsControls.js#L57, like this:

```js
{ isGA4Connected && (
	<h4 className="googlesitekit-settings-module__fields-group-title">
		{ __( 'Universal Analytics', 'google-site-kit' ) }
	</h4>
) }

<div className="googlesitekit-setup-module__inputs">
	<AccountSelect hasModuleAccess={ hasModuleAccess } />
	<PropertySelect hasModuleAccess={ hasModuleAccess } />
	<ProfileSelect hasModuleAccess={ hasModuleAccess } />
</div>
```

That would keep it in-line with the GA4 header below and look like this:

<img width="1156" alt="CleanShot 2023-03-22 at 12 20 04" src="https://user-images.githubusercontent.com/90871/226903158-51b0e945-1f4b-4d60-8492-e999ed24eb14.png">

Though some margins on the select elements still need tweaking.

Reply:
Hey @tofumatt, thanks for checking this but if you refer to the [Figma design](https://www.figma.com/file/0UCIE3hPeiu4NYdpAODASd/GA4-Updates?node-id=1-3&t=BpFQPtqaVuXmoig7-0), you'll see that this label is not expected for the usual Settings edit case:

![image](https://user-images.githubusercontent.com/18395600/226904438-22f31bf3-7e6e-4ad6-8f28-0562b7e2bdd4.png)

It's only there for the version where GA4 isn't set up yet:

![image](https://user-images.githubusercontent.com/18395600/226904765-2411f3e9-cd75-4244-ac0b-1e0417cc2590.png)

As a side note, as per this point in the QAB I've not tried to exactly match the design as it's out of sync with the current implementation.

> Note that the Figma design is not aligned with the current settings form/view, so please assess the implementation as aiming for consistency with the current view rather than trying to provide a direct match for what's in Figma.